src/components/outline-nav/utils/highlight-active-items.ts

```diff
@@ -0,0 +1,25 @@
+import { OutlineLinkItem } from '../types'
+
+/**
+ * Given an array of outline items, each of which has a `url` property,
+ * and a string representing #hash portion of the current URL,
+ *
+ * Return the array of items with `isActive` property set to `true` on any
+ * items that have a matching URL.
+ */
+function highlightActiveItems(
```

Comment:
🙋🏻 Having trouble picturing it at the moment, when are there multiple active items?

Reply:
We don't expect there to be multiple active items. I suppose if we wanted to we could add some logic here to try to exit early once we've found the first active item, but personally I felt this might not be worth the tradeoff in additional logic & complexity.